src/card/aggregate-status/aggregate-status-card.component.js

```diff
@@ -63,13 +66,19 @@
        <i>(depreciated, use layout = 'tall' instead)</i>
        </br></br>
        <pf-aggregate-status-card status="aggStatusAlt" show-top-border="true" alt-layout="true"></pf-aggregate-status-card>
+       <br/>
+       <label>Loading State</label>
+       <pf-aggregate-status-card status="aggStatusAlt2" loading-card-height="150px" show-top-border="true" show-spinner="dataLoading" spinner-text="Loading" layout="tall"></pf-aggregate-status-card>
```

Comment:
Looking around at other examples of this in angular-patternfly we do not include the 'px' as part of the string, just the number.

Reply:
Can remove :)